clog/clog.go

```diff
@@ -40,6 +40,18 @@ func (l *log) log(msg string, sev logger.Severity) {
 	logger.Log(logger.LogEntry{Message: msg, Severity: sev, CallDepth: 3, Labels: l.labels})
 }
 
+func (l *log) logStructured(structuredPayload interface{}, msg string, sev logger.Severity) {
+	// Set CallDepth 3, one for logger.Log, one for this function, and one for
+	// the calling clog function.
+	logger.Log(logger.LogEntry{Message: msg, StructuredPayload: structuredPayload, Severity: sev, CallDepth: 3, Labels: l.labels})
```

Comment:
Can we instead just use log (add structuredPayload) and update all the existing functions to just pass in nil that way we don't have two functions calling logger.Log

Reply:
Done.